packages/cli/commands/project/cloneApp.js

```diff
@@ -119,10 +119,15 @@ exports.handler = async options => {
 
       // Extract zipped app files and place them in correct directory
       const zippedApp = await downloadClonedProject(accountId, exportId);
-      await extractZipArchive(zippedApp, name, absoluteDestPath, {
-        includesRootDir: true,
-        hideLogs: true,
-      });
+      await extractZipArchive(
+        zippedApp,
+        sanitizeFileName(name),
```

Comment:
This isn't blocking, but I wonder if we should move this to the LDL side after we ship this and unblock the beta participant.

Reply:
You read my mind.